pages.th/windows/chkdsk.md

```diff
@@ -0,0 +1,20 @@
+# chkdsk
+
+> ความสมบูรณ์ของระบบไฟล์และข้อมูลเมตาของระบบไฟล์บนดิสก์โวลุ่มและแก้ไขข้อผิดพลาดของระบบ
+> ข้อมูลเพิ่มเติม: <https://learn.microsoft.com/windows-server/administration/windows-commands/chkdsk>.
+
+- ระบุตัวอักษรไดรฟ์ (ตามด้วยเครื่องหมาย colon), mount point, หรือชื่อของไดรฟ์:
+
+`chkdsk {{volume}}`
```

Comment:
Please translate the placeholders, like `size` and `volume`, for get this easier for non-English speakers.